src/main/resources/service_config.json

```diff
@@ -218,6 +218,29 @@
         "CREATE",
         "READ",
         "UPDATE",
-        "DELETE"]
-      }]
+        "DELETE"
+      ]
+    },
+    {
+      "id": "adoption_cos_api",
```

Comment:
Would we not ads the service id to here (or on flux)?
https://github.com/hmcts/ccd-case-document-am-api/blob/master/charts/ccd-case-document-am-api/values.yaml#L18

Reply:
this is done